src/Vault.ts

```diff
@@ -1,60 +1,9 @@
 import { Account } from './Account';
 import { masterVault as backend } from '@xbacked-dao/xbacked-contracts';
+import { VaultReturnParams, ReachUserVault, UserVaultReturnParams, VaultParameters } from './interfaces';
```

Comment:
This class should be changed to use the ClassName `Contract` since the meaning of the word `Vault` has changed for a while now, i can implement this in my own P.R if you want.